src/main/scala/com/microsoft/partnercatalyst/fortis/spark/ProjectFortis.scala

```diff
@@ -0,0 +1,194 @@
+
+    // TODO: other computations and save to DB
+    fortisEvents.foreach(
+      _.foreachRDD(
+        _.foreachPartition(throw new NotImplementedException)))
+
+    streamingContext
+  }
+
+  // Main starts here
+  val ssc = StreamingContext.getOrCreate(Settings.progressDir, createStreamingContext)
+
+  ssc.start()
+  ssc.awaitTermination()
+
+  /**
+    * TODO: Use Cassandra to populate
+    * Build connector config registry from hard-coded values for demo.
+    *
+    * The key is the name of the pipeline and the value is a list of connector configs whose streams should comprise it.
+    */
+  private def getStreamRegistry: Map[String, List[ConnectorConfig]] = {
+    Map[String, List[ConnectorConfig]](
+      "instagram" -> List(
+        ConnectorConfig(
+          "InstagramTag",
+          Map(
+            "authToken" -> System.getenv("INSTAGRAM_AUTH_TOKEN"),
+            "tag" -> "rose"
+          )
+        ),
+        ConnectorConfig(
+          "InstagramLocation",
+          Map(
+            "authToken" -> System.getenv("INSTAGRAM_AUTH_TOKEN"),
+            "latitude" -> "49.25",
+            "longitude" -> "-123.1"
+          )
+        )
+      ),
+      "radio" -> List(
+        ConnectorConfig(
+          "Radio",
+          Map(
+            "subscriptionKey" -> System.getenv("OXFORD_SPEECH_TOKEN"),
+            "radioUrl" -> "http://live02.rfi.fr/rfimonde-32.mp3", // "http://bbcmedia.ic.llnwd.net/stream/bbcmedia_radio3_mf_p",
+            "audioType" -> ".mp3",
+            "locale" -> "fr-FR", //"en-US",
+            "speechType" -> "CONVERSATION",
+            "outputFormat" -> "SIMPLE"
+          )
+        )
+      ),
+      "twitter" -> List(
+        ConnectorConfig(
+          "Twitter",
+          Map(
+            "consumerKey" -> System.getenv("TWITTER_CONSUMER_KEY"),
+            "consumerSecret" -> System.getenv("TWITTER_CONSUMER_SECRET"),
+            "accessToken" -> System.getenv("TWITTER_ACCESS_TOKEN"),
+            "accessTokenSecret" -> System.getenv("TWITTER_ACCESS_TOKEN_SECRET")
+          )
+        )
+      ),
+      "facebook" -> List(
+        ConnectorConfig(
+          "FacebookPage",
+          Map(
+            "accessToken" -> System.getenv("FACEBOOK_AUTH_TOKEN"),
+            "appId" -> System.getenv("FACEBOOK_APP_ID"),
+            "appSecret" -> System.getenv("FACEBOOK_APP_SECRET"),
+            "pageId" -> "aljazeera"
+          )
+        )
+      ),
+      "tadaweb" -> List(
+        ConnectorConfig(
+          "Tadaweb",
+          Map (
+            "policyName" -> System.getenv("TADAWEB_EH_POLICY_NAME"),
+            "policyKey" -> System.getenv("TADAWEB_EH_POLICY_KEY"),
+            "namespace" -> System.getenv("TADAWEB_EH_NAMESPACE"),
+            "name" -> System.getenv("TADAWEB_EH_NAME"),
+            "partitionCount" -> System.getenv("TADAWEB_EH_PARTITION_COUNT"),
+            "consumerGroup" -> "$Default"
+          )
+        )
+      )
+    )
+  }
+
+  private object DriverLog {
```

Comment:
I'd agree that System.err would be a better choice, however we need the logs to go up to AppInsights, right? Will that still work without using log4j? Happy to remove this if so.